clusterloader2/pkg/prometheus/manifests/default/master-ip/master-serviceMonitor.yaml

```diff
@@ -15,8 +15,9 @@ spec:
     port: etcd
   {{end}}
   {{if $PROMETHEUS_SCRAPE_NODE_EXPORTER}}
-  - interval: 5s
+  - interval: 30s
```

Comment:
Please add TODO to see if we could disable some collectors in node-exporter and keep the scrape interval at 5s (consistent with other scrape intervals)

Reply:
DONE